streamly.cabal

```diff
@@ -249,9 +249,17 @@ common exe-options
   import: compile-options, optimization-options, threading-options
 
 -- Some benchmarks are threaded some are not
+-- XXX dependencies should be separated under bench-depends
 common bench-options
   import: compile-options, optimization-options
-  ghc-options: -with-rtsopts "-T"
+  ghc-options: -with-rtsopts "-T -M256M"
```

Comment:
Although not required, it would me nice to document the problem that lead to the use of limited memory and mention it here or a small writeup would be sufficient.
Or just leave it be, I'll document it later.

Reply:
Never mind this. I just noticed that you've written some of it in the commit message.